package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "integreatly-web-app",
-  "version": "2.26.0",
+  "version": "2.26.1",
   "private": true,
   "proxy": "http://localhost:5001/",
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.7.2",
-    "@patternfly/patternfly": "2.43.1",
-    "@patternfly/react-core": "3.124.1",
-    "@patternfly/react-icons": "^3.14.23",
+    "@patternfly/patternfly": "4.10.29",
+    "@patternfly/react-core": "4.18.1",
```

Comment:
can we bump to `4.18.13`?

Reply:
I've updated to the latest PF:
patternfly: 4.16.7
react-core: 4.23.1